easybuild/tools/config.py

```diff
@@ -65,7 +66,7 @@
 }
 DEFAULT_PREFIX = os.path.join(os.path.expanduser('~'), ".local", "easybuild")
 DEFAULT_REPOSITORY = 'FileRepository'
-
+DEFAULT_PACKAGE_TEMPLATE = "eb-%(name)s-%(version)s-%(toolchain)s"
```

Comment:
oops, this is `config.py` indeed. Where is the cancel button for the above?! -P
Anyway, the need for potential suffix still applies, but is not as urgent, it can be done anytime later.